openloops.spec

```diff
@@ -24,6 +24,7 @@ EOF
 export SCONSFLAGS="-j %{compiling_processes}"
 ./openloops update --processes generator=0
 ./openloops libinstall lhc.coll
+./openloops libinstall %{_sourcedir}/openloops-user.coll
```

Comment:
@SiewYan is the line above still needed?

Reply:
No, you may remove 

./openloops libinstall lhc.coll